File: chaerim/Programmers/Level_2/Lv2_구명보트.py
def solution(people, limit):
    answer = 0

    people.sort()

    light_people = 0
    heavy_people = len(people)-1
    while light_people <= heavy_people:
        if people[light_people] + people[heavy_people] <= limit:
            light_people += 1
            heavy_people -= 1
            answer += 1
        else:
            heavy_people -= 1
            answer += 1

    return answer

File: seongwon/Programmers/Level_2/Lv2_구명보트.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

int solution(vector<int> people, int limit) {
    int answer = 0;
    sort(people.begin(), people.end());
    int i = 0, j = people.size()-1;
    
    while(i <= j){
        if(people[i] + people[j] <= limit){
            i++;
        }
        j--;
        answer++;
        people.pop_back();
    }
    return answer;
}
